src/js/components/generateDetachedFiles/GenerateDetachedFilesPage.jsx

```diff
@@ -308,7 +313,8 @@ export default class GenerateDetachedFilesPage extends React.Component {
                 handleDateChange={this.handleDateChange.bind(this)}
                 updateError={this.updateError.bind(this)}
                 generateFile={this.generateFile.bind(this)}
-                toggleAgencyType={this.toggleAgencyType} />);
+                toggleAgencyType={this.toggleAgencyType}
+                clickedDownload={this.clickedDownload.bind(this)} />);
```

Comment:
My main concern is binding in the prop. `This is bad for performance, as it may cause unnecessary re-renders if a brand new function is passed as a prop to a component that uses reference equality check on the prop to determine if it should update.` But after looking throughout the application it looks like we are binding in the prop a lot of places.